### PR TITLE
.NET Aspire 8.1 Samples Updates

### PR DESCRIPTION
**Pull request by mitchdenny**
This PR is gathering up all the .NET Aspire 8.1 sample updates into a single PR so we can merge them as a batch.

### CODE REVIEW

File: NuGet.config [hunk 1]
```diff
@@ -3,12 +3,16 @@
   <packageSources>
     <clear />
     <add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
+    <add key="dotnet8feed" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet8/nuget/v3/index.json" protocolVersion="3" />
```
Comment (danmoseley) [inline NuGet.config:6]: We presumably want to remove this as soon as 8.1 is out.

**Review by danmoseley — APPROVED** [review]
This assumes we want this in main. I see 3 options
1. just validate in the PR, and only merge when 8.1 goes out (but removing the nuget.config line). this is what we've done in eshop, eg.. 
2. merge into main, remove the nuget.config line when 8.1 goes out. downside: most people consuming aspire probably want to start with stable builds.
3. merge into release/v.next type branch. not sure the value of this and it would need infra setting up.

I'm assuming you're planning # 1?

File: NuGet.config (hunk 1 above)
Comment (mitchdenny) [reply NuGet.config:6]: Yes we will remove this once 8.1 ships and we merge this PR (as well as updating all the package versions).

**Comment by danmoseley** [comment]
> Please how can ii contribute

Hi @dom3691 welcome. What kind of contribution are you looking to make? There are some open issues in this repo you can look at. Perhaps you might want to look at Aspire proper -- see https://github.com/dotnet/aspire/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22 and find one that interests you, then make a comment on that.

**Comment by danmoseley** [comment]
eg., this one looks like a potentially good first issue
https://github.com/dotnet/aspire/issues/2649
if you did this, I would make a single change (just one fix in one place) and put up a PR to check you're doing the kind of thing expected, before doing more.

**Comment by mitchdenny** [comment]
> I'm assuming you're planning # 1?

Yes. Thats my plan.

File: .gitattributes [hunk 2]
```diff
@@ -3,6 +3,8 @@
 ###############################################################################
 * text=auto
 *.sh text eol=lf
+*.mod binary
```
Comment (eerhardt) [inline .gitattributes:6]: @mitchdenny - what is this change for?

File: tests/SamplesIntegrationTests/AppHostTests.cs [hunk 3]
```diff
@@ -97,7 +97,7 @@ public async Task TestEndpointsReturnOk(TestEndpoints testEndpoints)
             }
         }
 
-        appHostLogs.EnsureNoErrors();
+        // DCP is now sending stderr?? appHostLogs.EnsureNoErrors();
```
Comment (eerhardt) [inline tests/SamplesIntegrationTests/AppHostTests.cs:100]: We should not merge with this change.

**Review by eerhardt — DISMISSED** [review]

File: .gitattributes (hunk 2 above)
Comment (danmoseley) [reply .gitattributes:6]: I'm guessing git is mis-detecting those go files as text, and therefore mangling the line endings (which are actually not line endings)

Comment (danmoseley) [reply .gitattributes:6]: oh, hold on, they are apparently text. which is I guess your point.. Maybe go complains if they have \r\n

Comment (mitchdenny) [reply .gitattributes:6]: Exactly. It kept getting them wrong.

Comment (eerhardt) [reply .gitattributes:6]: I'm not sure telling git that these are binary is the right call.

Looking at other .gitattributes:

https://grep.app/search?q=%2A.mod&words=true&filter[lang][0]=Git%20Attributes

People mostly tell it that `eol=lf`.

**Comment by radical** [comment]
Using [linux-dev-certs](https://github.com/tmds/linux-dev-certs) tool seems to fix the failures.

File: tests/SamplesIntegrationTests/AppHostTests.cs (hunk 3 above)
Comment (eerhardt) [reply tests/SamplesIntegrationTests/AppHostTests.cs]: This has been resolved in the latest change.

**Comment by mitchdenny** [comment]
Merging thanks for resolving this @eerhardt

**Pull request merged**